sidebars.js

```diff
@@ -448,6 +448,26 @@ module.exports = {
         "references/events",
         "references/configuration",
         "references/ui-configuration",
+        {
+          type: "link",
+          label: "Go SDK API",
+          href: "https://pkg.go.dev/go.temporal.io/sdk",
+        },
+        {
+          type: "link",
+          label: "Java SDK API",
+          href: "https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/index.html",
+        },
+        {
+          type: "link",
+          label: "Pythin SDK API",
```

Comment:
```suggestion
          label: "Python SDK API",
```
"Pythin" is the nickname of "Python Lite," IIRC. 😁